sqlx-core/src/mysql/options.rs

```diff
@@ -152,6 +155,25 @@ impl MySqlConnectOptions {
         self
     }
 
+    /// Sets a custom path to a socket, switching the connection method from TCP
+    /// to a unix domain socket.
```

Comment:
Nit: "socket" commonly refers to both UDS and TCP. Perhaps may be clearer if written as:

```suggestion
    /// Pass a path to a Unix socket. This changes the connection stream from TCP to UDS.
```